samples/drivers/flash_sim/sample.yaml

```diff
@@ -0,0 +1,11 @@
+sample:
+  name: Flash simulator
+tests:
+  test:
+    platform_whitelist: qemu_x86 qemu_cortex_m3
+    tags: simulator
+    harness: console
+    harness_config:
+      type: one_line
+      regex:
+        - "All tests completed."
```

Comment:
new line

Reply:
You mean I should have a newline at the end of this file or a newline in the string?
It's a regexp so just a substring should be okay for a match.

Reply:
EOF